extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -295,13 +304,20 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     if (vpRange === false) {
       return vpRange;
     }
+    const renderOutsideViewport = this.renderOutsideViewport();
+    // False will occur when throttle in effect.
+    if (!!this.experimentIds.filter(exp =>
```

Comment:
can't you just use this.experimentIds.includes() here instead? using a filter and then checking the length seems overly complicated